server/src/controllers/authController.ts

```diff
@@ -116,17 +116,12 @@ export const authorizationCodePopupHandler = async (
   // console.log("response.tokens", response.tokens);
 
   // get the ACCESS TOKEN, REFRESH TOKEN, ID TOKEN, and Token Expiry Date
-  const {
-    access_token: accessToken,
-    refresh_token: refreshToken,
-    id_token: idToken,
-    expiry_date: tokenExpiryDate,
-  } = response.tokens;
+  const { id_token: idToken } = response.tokens;
```

Comment:
I would like to keep these in here but commented out
So that when i am adjusting the Auth in future i don't have to remember these

```suggestion
  const {
    // access_token: accessToken,
    // refresh_token: refreshToken,
    id_token: idToken,
    // expiry_date: tokenExpiryDate,
  } = response.tokens;
```